lib/datadog/core/configuration/settings.rb

```diff
@@ -450,19 +450,6 @@ def initialize(*_)
               o.env 'DD_PROFILING_UPLOAD_PERIOD'
               o.default 60
             end
-
-            # Enables reporting of information when the Ruby VM crashes.
-            #
-            # This feature is no longer experimental, and we plan to deprecate this setting and replace it with a
-            # properly-named one soon.
-            #
-            # @default `DD_PROFILING_EXPERIMENTAL_CRASH_TRACKING_ENABLED` environment variable as a boolean,
-            # otherwise `true`
-            option :experimental_crash_tracking_enabled do |o|
-              o.type :bool
-              o.env 'DD_PROFILING_EXPERIMENTAL_CRASH_TRACKING_ENABLED'
-              o.default true
-            end
           end
```

Comment:
While this option is experimental, removing it is still a breaking change. I suggest replacing it with a deprecation message (similar to the ones we did in the 1.x-stable) branch.